src/data/nat/digits.lean

```diff
@@ -236,10 +238,71 @@ begin
   { simp [of_digits, list.sum_cons, ih], }
 end
 
-/-! ### Inequalities
+/-!
+### Properties
+
+This section contains various lemmas of properties relating to `digits` and `of_digits`.
+-/
+
+lemma digits_eq_nil_iff_eq_zero (b n : ℕ) : digits b n = [] ↔ n = 0 :=
```

Comment:
```suggestion
lemma digits_eq_nil_iff_eq_zero {b n : ℕ} : digits b n = [] ↔ n = 0 :=
```
We usually make arguments of `iff` lemmas implicit, so that it's easier to use projection notation.